backend.py

```diff
@@ -518,7 +518,11 @@ def configureNTP(mounts, ntp_config_method, ntp_servers):
     util.runCmd2(['chroot', mounts['root'], 'systemctl', 'enable', 'chronyd'])
     util.runCmd2(['chroot', mounts['root'], 'systemctl', 'enable', 'chrony-wait'])
 
-def inspectTargetDisk(disk, existing, preserve_first_partition, create_sr_part):
+# This is attempting to understand the desired layout of the future partitioning
+# based on options passed and status of disk (like partition to retain).
+# This is used on upgrade or restore.
```

Comment:
`This should be used for upgrade or install, not for restore.`

Reply:
Done